src/app/core/core.component.ts

```diff
@@ -52,4 +54,37 @@ export class CoreComponent implements OnInit, OnDestroy {
   isSolicitor(): boolean {
     return this.profile.isSolicitor();
   }
+
+  isUnsupportedBrowser(): boolean {
+
+    let ua = window.navigator.userAgent;
```

Comment:
using `window` global object directly in Angular is not recommended. Any way we could avoid this?

Reply:
for example have you considered using an angular library for device info detection: e.g.
https://github.com/KoderLabs/ngx-device-detector

Reply:
Done the change.  Thank you.